### PR TITLE
VBS: VVjj semi-leptonic cards

### PR DESCRIPTION
**Pull request by arapyan**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/VVjj_semileptonic/aQGC/runjobs.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+samples=("WMhadZlepJJ_EWK_LO_SM_mjj100_pTj10" "WMlepWMhadJJ_EWK_LO_SM_mjj100_pTj10" "WMlepZhadJJ_EWK_LO_SM_mjj100_pTj10" "WPhadWMlepJJ_EWK_LO_SM_mjj100_pTj10" "WPhadZlepJJ_EWK_LO_SM_mjj100_pTj10" "WPlepWMhadJJ_EWK_LO_SM_mjj100_pTj10" "WPlepWPhadJJ_EWK_LO_SM_mjj100_pTj10" "WPlepZhadJJ_EWK_LO_SM_mjj100_pTj10" "ZlepZhadJJ_EWK_LO_SM_mjj100_pTj10")
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/VVjj_semileptonic/aQGC/runjobs.sh]: do we really need this file?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/VVjj_semileptonic/EWK/WMhadZlepJJ_EWK_LO_SM_mjj100_pTj10/WMhadZlepJJ_EWK_LO_SM_mjj100_pTj10_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#* The MadGraph5_aMC@NLO Development Team - Find us at *
+#* https://server06.fynu.ucl.ac.be/projects/madgraph *
+#* *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+#initialization parameters
+#set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/VBS/VVjj_semileptonic/EWK/WMhadZlepJJ_EWK_LO_SM_mjj100_pTj10/WMhadZlepJJ_EWK_LO_SM_mjj100_pTj10_madspin_card.dat]: can you please cleanup the datacards removing commented settings?

**Comment by arapyan** [comment]
Ciao Luca, your comments were implemented.

**Comment by perrozzi** [comment]
things look ok to me, merging

**Pull request merged**